### ISSUE TITLE
RUPS does not display any custom/private keys that are in the trailer dictionary

### ISSUE THREAD

**Issue by petervwyatt** [itext/rups#33]
Thanks iText for opening up this bug list!

RUPS does not display any custom/private keys that are in the trailer dictionary.

See this PDF: https://s3.documentcloud.org/documents/23108357/redacted-version-of-exhibits-a-j-to-letter-to-the-honorable-kathaleen-st-j-mccormick-from-edward-b.pdf
The `/Source` key in the (conventional) trailer dictionary is not shown.

**Comment by MatthiasValvekens** [comment]
I don't think RUPS currently displays anything other than ID, Root and Info...

@michaeldemey Didn't you tell me in Berlin that you had a patch for that? Maybe I'm confusing it with another conversation.

**Comment by michaeldemey** [comment]
Nope, we didn't know of this in Berlin. @petervwyatt brought it up two weeks ago. This has more to do with how iText core deals with stamping. I have a fix for it on a local branch which needs to update a lot of tests to account for the updated /Trailer behaviour.

**Comment by Eswcvlad** [comment]
Looks like this, actually, got fixed in `25.03`. And, what is more surprising, the `/Source` key is present in both read-only and read-write modes.

General trailer modification issues are still present in read-write mode, but I'll leave those to #154.
